### ISSUE TITLE
DIA-NN exited without saving the files in the indicated location

### ISSUE THREAD

**Issue by mrosario01** [vdemichev/DiaNN#1899]
Hi,  How are you?

I have a problem with MSe files. (I used MsConvert to save  mzML pea picking vendor)
DIANN version 2.3.2 Academy
With a previously generated spectral library with DIANN

DIA-NN exited but it did not generate any of the files it reported creating. According to the messages the run seemed to finish successfully, but no output documents were produced. This is my second attempt.

Another issue is that the processing takes more than 7 hours and still does not generate the results.

My files are DIA MSe data acquired on a Waters instrument, and the analysis includes ubiquitination.

....
[#0]: 5.95515e-07, 0.04044, 0.396822, 0.330466, 0, 0.0100476, 0.111521, 1, [644:28] Calculating q-values [655:07] Number of IDs at 50%, 5%, 1%, 0.1% FDR: 21, 2, 0, 0 [655:07] Calibrating retention times [656:18] 2 precursors used for iRT estimation. [656:18] Processing batches #84-100 out of 200 

[656:18] Precursor search DIA-NN exited diann-stats.exe "D:\LACTAD\DIANN\report.parquet" PDF report will be generated in the background
.......

**Comment by vdemichev** [comment]
Hi,

Can you please share the full log?

Best,
Vadim

**Comment by mrosario01** [comment]
Sorry, I didn’t save the log, but I’m generating it again (now with only two files). The beginning is the same as the previous one, and it looked like this:

diann.exe --f "D:\LACTAD\Musculo\Animais_Controle\3_Processamento\mzml_vendor\20250227_AnaLaura_FCA_KO.mzML
" --f "D:\LACTAD\Musculo\Animais_Controle\3_Processamento\mzml_vendor\20250227_AnaLaura_FCA_WT.mzML
" --lib "D:\LACTAD\Musculo\Fasta\library diann\MusMu_LibSpec_050326.predicted.speclib" --threads 10 --verbose 1 --out "D:\LACTAD\DIANN\report.parquet" --qvalue 0.01 --matrices  --min-corr 2.0 --time-corr-only --extracted-ms1 --min-cal 500 --min-class 1000 --pre-filter --xic --fasta "D:\LACTAD\Musculo\Fasta\UP000000589_10090.fasta" --met-excision --min-pep-len 7 --max-pep-len 50 --min-pr-mz 300 --max-pr-mz 1800 --min-pr-charge 1 --max-pr-charge 4 --min-fr-mz 200 --max-fr-mz 1800 --cut K*,R* --missed-cleavages 2 --unimod4 --var-mods 2 --var-mod UniMod:121,114.042927,K --no-cut-after-mod UniMod:121 --proteoforms --reanalyse --rt-profiling  

DIA-NN 2.3.2 Academia  (Data-Independent Acquisition by Neural Networks)
Compiled on Jan 22 2026 10:50:24
Current date and time: Tue Mar 10 13:35:33 2026
CPU: GenuineIntel Intel(R) Core(TM) 7 150U
SIMD instructions: AVX AVX2 FMA SSE4.1 SSE4.2 
Logical CPU cores: 12
13Gb out of 31Gb RAM is free
Thread number set to 10
Output will be filtered at 0.01 FDR
Precursor/protein x samples expression level matrices will be saved along with the main report
Only peak groups with correlation-based score exceeding 2 will be considered; will be deactivated for the second MBR pass
DIA-NN will disable machine learning during the calibration stage to save memory, this may lead to somewhat lower identification numbers; will be deactivated for the second MBR pass
Ultra-fast mode will be used, this may worsen the identification performance; will be deactivated for the second MBR pass
Minimum number of precursors used for calibration set to 500
Minimum number of precursors used for linear classifier training set to 1000
DIA-NN will pre-filter candidate precursors
XICs within 10 seconds from the apex will be extracted for each precursor and saved in .parquet format, a folder will be created next to the main report for the XICs storage
N-terminal methionine excision enabled
Min peptide length set to 7
Max peptide length set to 50
Min precursor m/z set to 300
Max precursor m/z set to 1800
Min precursor charge set to 1
Max precursor charge set to 4
Min fragment m/z set to 200
Max fragment m/z set to 1800
In silico digest will involve cuts at K*,R*
Maximum number of missed cleavages set to 2
Cysteine carbamidomethylation enabled as a fixed modification
Maximum number of variable modifications set to 2
Modification UniMod:121 with mass delta 114.043 at K will be considered as variable
Protein identification will rely on peptidoform confidence for matched peptides
MBR enabled; .quant files will only be saved to disk during the first pass
The spectral library (if generated) will retain the original spectra but will include empirically-aligned RTs
DIA-NN will automatically optimise the mass accuracy for the first run of the experiment, use this mode for preliminary analyses only
The following variable modifications will be localised: UniMod:121 
In silico cuts after the following modifications will not be considered: UniMod:121 

2 files will be processed
[0:00] Loading spectral library D:\LACTAD\Musculo\Fasta\library diann\MusMu_LibSpec_050326.predicted.speclib
[1:19] Library annotated with sequence database(s): D:\LACTAD\Musculo\Fasta\UP000000589_10090.fasta
[1:21] Spectral library loaded: 21606 protein isoforms, 26643 protein groups and 12011312 precursors in 4042359 elution groups.
[1:21] Loading protein annotations from FASTA D:\LACTAD\Musculo\Fasta\UP000000589_10090.fasta
[1:21] Annotating library proteins with information from the FASTA database
[1:21] Gene names missing for some isoforms
[1:21] Library contains 21606 proteins, and 21303 genes
[1:32] Initialising library

First pass: generating a spectral library from DIA data

[2:04] File #1/2
[2:04] Loading run D:\LACTAD\Musculo\Animais_Controle\3_Processamento\mzml_vendor\20250227_AnaLaura_FCA_KO.mzML
WARNING: check if this run is DDA - in this case use --dda to analyse it
[3:50] Analysing MS1 spectra
[3:50] Analysing MS2 spectra
[3:50] Integrating MS1 spectra
[4:34] Integrating MS2 spectra
[4:51] Pre-processing...
WARNING: MS2 mass calibration may be off by more than 15 ppm, consider examining the PDF report generated by DIA-NN to verify if this is the case.
[4:51] Detected MS2 mass shift based on the analysis of background ions: -45.1874 ppm
[4:57] 2602 MS1 and 2522 MS2 scans in 82 (inferred) and 82 (encoded) cycles, 12011312 precursors in range
[4:59] Calibrating with mass accuracies 30 (MS1), 56 (MS2)




****after many hours,**** 

[516:23] Optimising weights Averages: [#0]: -0.00112491, -0.0389486, -0.00423335, 0.00388297, -0.00455143, -0.00573142, -0.042962, -0.00151044, Weights: [#0]: 6.97704e-07, 0, 0, 1, 0, 0.0899756, 0, 0, [516:24] Calculating q-values [516:25] Number of IDs at 50%, 5%, 1%, 0.1% FDR: 17, 2, 2, 0 [516:25] Resetting weights [516:25] Precursor search [516:25] Optimising weights Averages: [#0]: -0.000290886, 0.052819, 0.00211882, -0.000649895, 0.000909041, -0.00523271, -0.0168267, -0.00107495, Weights: [#0]: 2.03364e-06, 0.180431, 1, 0, 0.200854, 0.103164, 0, 0, [516:26] Calculating q-values [516:27] Number of IDs at 50%, 5%, 1%, 0.1% FDR: 0, 0, 0, 0 [516:27] Calibrating retention times [516:27] Processing batches #18-21 out of 200 [516:27] Precursor search [519:58] Optimising weights Averages: [#0]: -0.000442853, 0.00651164, -0.00253871, 0.000686025, 0.00160475, 0.00232467, -0.00618516, -0.00149916, Weights: [#0]: 4.20526e-06, 0.0282367, 0, 1, 0.519035, 0.683876, 0, 0, [519:59] Calculating q-values [520:00] Number of IDs at 50%, 5%, 1%, 0.1% FDR: 8, 2, 0, 0 [520:00] Resetting weights [520:00] Precursor search [520:01] Optimising weights Averages: [#0]: -0.000108251, 0.0199811, -0.00111713, 0.00155413, 0.00169947, 0.00415597, 0.00201964, -0.000864918, Weights: [#0]: 1.80964e-06, 0.052708, 0.414429, 1, 0.220717, 0.233694, 0.0739679, 0, [520:01] Calculating q-values [520:03] Number of IDs at 50%, 5%, 1%, 0.1% FDR: 12, 4, 2, 0 [520:03] Calibrating retention times [520:03] 4 precursors used for iRT estimation. [520:03] Processing batches #22-26 out of 200 [520:03] Precursor search [524:25] Optimising weights Averages: [#0]: 0.000214035, 0.0546431, -0.00161566, -0.00119447, 0.00410471, 0.0177546, 0.00839931, -0.000988926, Weights: [#0]: 9.6378e-07, 0.0712374, 1, 0, 0.346115, 0.546575, 0.0597652, 0.40073, [524:26] Calculating q-values [524:28] Number of IDs at 50%, 5%, 1%, 0.1% FDR: 5, 0, 0, 0 [524:28] Resetting weights [524:28] Precursor search [524:29] Optimising weights Averages: [#0]: 0.000168167, 0.0336651, -0.00312009, -0.00125291, 0.00561392, 0.0165496, 0.00614134, -0.000633665, Weights: [#0]: 2.4301e-06, 0.108802, 0.545565, 0, 1, 0.869133, 0, 0, [524:30] Calculating q-values [524:31] Number of IDs at 50%, 5%, 1%, 0.1% FDR: 4, 0, 0, 0 [524:31] Calibrating retention times [524:31] 2 precursors used for iRT estimation. [524:31] Processing batches #27-32 out of 200 [524:31] Precursor search [529:49] Optimising weights Averages: [#0]: 0.00040222, 0.0256169, 0.000901399, -0.00234825, 0.00341142, 0.0123359, -0.000391546, -0.00147556, Weights: [#0]: 1.054e-06, 0.0322195, 1, 0, 0.317245, 0.463913, 0, 0, [529:51] Calculating q-values [529:53] Number of IDs at 50%, 5%, 1%, 0.1% FDR: 5, 0, 0, 0 [529:53] Resetting weights [529:53] Precursor search [529:54] Optimising weights Averages: [#0]: 0.000128012, 0.0193083, -0.00215146, -0.00291855, 0.00368997, 0.0131786, 0.00686462, -0.00119726, Weights: [#0]: 2.98948e-06, 0.0650941, 1, 0, 0.823871, 0.933067, 0.0808864, 0, [529:55] Calculating q-values [529:57] Number of IDs at 50%, 5%, 1%, 0.1% FDR: 5, 0, 0, 0 [529:57] Calibrating retention times [529:57] 2 precursors used for iRT estimation. [529:57] Processing batches #33-39 out of 200 [529:57] Precursor search [536:06] Optimising weights Averages: [#0]: 0.000419354, 0.0273381, 0.000466137, -0.000804166, 0.00518907, 0.0135755, 0.00252421, -0.00028389, Weights: [#0]: 6.64319e-07, 0.0214678, 0.882066, 0, 0.275153, 0.406221, 0, 1, [536:08] Calculating q-values [536:11] Number of IDs at 50%, 5%, 1%, 0.1% FDR: 4, 0, 0, 0 [536:11] Resetting weights [536:11] Precursor search [536:12] Optimising weights Averages: [#0]: 0.000596107, 0.0401044, -0.00162489, -0.00217068, 0.00455291, 0.0169791, 0.0102036, 0.000270047, Weights: [#0]: 7.9864e-07, 0.0410182, 0.464621, 0, 0.267852, 0.313993, 0, 1, [536:13] Calculating q-values [536:15] Number of IDs at 50%, 5%, 1%, 0.1% FDR: 6, 0, 0, 0 [536:15] Calibrating retention times [536:15] Processing batches #40-47 out of 200 [536:15] Precursor search [543:16] Optimising weights Averages: [#0]: 0.00115057, 0.0093242, 0.000648045, 0.00106919, -5.055e-05, 0.020166, 0.0111175, 0.000703154, Weights: [#0]: 3.71263e-07, 0.000795025, 0.553451, 0.0960648, 0.0262429, 0.250793, 0, 1, [543:19] Calculating q-values [543:21] Number of IDs at 50%, 5%, 1%, 0.1% FDR: 6, 1, 0, 0 [543:21] Resetting weights [543:21] Precursor search [543:23] Optimising weights Averages: [#0]: 0.00107712, 0.00972365, -0.00169692, 0.000880159, 0.00199635, 0.0224108, 0.01701, 0.0011758, Weights: [#0]: 3.81978e-07, 0.00125537, 0.331006, 0.0384513, 0.0590764, 0.20449, 0, 1, [543:24] Calculating q-values [543:27] Number of IDs at 50%, 5%, 1%, 0.1% FDR: 6, 0, 0, 0 [543:27] Calibrating retention times [543:27] 2 precursors used for iRT estimation. [543:27] Processing batches #48-57 out of 200 [543:27] Precursor search [552:47] Optimising weights Averages: [#0]: 0.000984045, 0.0192808, 0.00011191, 0.00225137, 0.00204097, 0.0181613, 0.0197318, 0.000469303, Weights: [#0]: 4.50025e-07, 0.00833175, 0.84384, 0.330211, 0.0751817, 0.282525, 0.109861, 1, [555:28] Calculating q-values [556:13] Number of IDs at 50%, 5%, 1%, 0.1% FDR: 22, 2, 0, 0 [556:13] Resetting weights [556:13] Precursor search [556:56] Optimising weights Averages: [#0]: 0.000512234, 0.027509, 0.00057049, 0.00218057, 0.0026323, 0.0119052, 0.0140789, 0.000366696, Weights: [#0]: 4.88654e-07, 0.0166495, 0.948874, 0.365406, 0.0875443, 0.259353, 0.144795, 1, [559:45] Calculating q-values [561:11] Number of IDs at 50%, 5%, 1%, 0.1% FDR: 15, 2, 0, 0 [561:11] Calibrating retention times [561:30] 2 precursors used for iRT estimation. [561:30] Processing batches #58-69 out of 200 [561:30] Precursor search [573:10] Optimising weights Averages: [#0]: 0.000811378, 0.00531206, 0.000885584, 0.00193423, 0.00398712, 0.00488036, 0.0216937, 0.00161415, Weights: [#0]: 3.63029e-07, 0.00336653, 0.614742, 0.194005, 0.0768056, 0.0950051, 0.142932, 1, [579:57] Calculating q-values [588:59] Number of IDs at 50%, 5%, 1%, 0.1% FDR: 22, 2, 0, 0 [588:59] Resetting weights [588:59] Precursor search [590:42] Optimising weights Averages: [#0]: 0.00068335, 0.0195994, -0.000219386, 0.00110011, 0.00444317, 0.00475564, 0.0221599, 0.00126915, Weights: [#0]: 6.29924e-07, 0.0183833, 0.664049, 0.177957, 0.144158, 0.0460857, 0.244131, 1, [594:22] Calculating q-values [602:01] Number of IDs at 50%, 5%, 1%, 0.1% FDR: 5, 2, 0, 0 [602:01] Calibrating retention times [602:48] 2 precursors used for iRT estimation. [602:48] Processing batches #70-83 out of 200 [602:48] Precursor search [617:53] Optimising weights Averages: [#0]: 0.00107126, 0.0305416, 0.00139349, 0.00110259, -0.00277678, 0.00976904, 0.0132415, 0.0017241, Weights: [#0]: 3.06716e-07, 0.0127923, 0.382834, 0.072432, 0, 0.108218, 0.0206197, 1, [624:15] Calculating q-values [635:30] Number of IDs at 50%, 5%, 1%, 0.1% FDR: 3, 0, 0, 0 [635:30] Resetting weights [635:30] Precursor search [638:22] Optimising weights Averages: [#0]: 0.00104961, 0.0482664, 0.000645387, 0.00191278, -0.00206093, 0.00730826, 0.019398, 0.0014858, Weights: [#0]: 5.95515e-07, 0.04044, 0.396822, 0.330466, 0, 0.0100476, 0.111521, 1, [644:28] Calculating q-values [655:07] Number of IDs at 50%, 5%, 1%, 0.1% FDR: 21, 2, 0, 0 [655:07] Calibrating retention times [656:18] 2 precursors used for iRT estimation. [656:18] Processing batches #84-100 out of 200 [656:18] Precursor search **DIA-NN exited diann-stats.exe "D:\LACTAD\DIANN\report.parquet" PDF report will be generated in the background**

**Comment by vdemichev** [comment]
> WARNING: check if this run is DDA - in this case use --dda to analyse it

If it's not really DDA, then it's likely not reading the file correctly. 

> 13Gb out of 31Gb RAM is free

Almost certainly out of RAM. You can also try InfinDIA (but do fix all mass accuracies & scan window), but I would not have high hopes that this is going to work. So for the first try (not to wait for long), makes sense to try some public library. Again, would not really expect this to work, but there's a chance it will.

> [635:30] Precursor search [638:22] Optimising weights Averages: [#0]: 0.00104961, 0.0482664, 0.000645387, 0.00191278, -0.00206093, 0.00730826, 0.019398, 0.0014858, Weights: [#0]: 5.95515e-07, 0.04044, 0.396822, 0.330466, 0, 0.0100476, 0.111521, 1, [644:28] Calculating q-values [655:07] Number of IDs at 50%, 5%, 1%, 0.1% FDR: 21, 2, 0, 0

Means it's completely unable to find anything in the data.

**Comment by mrosario01** [comment]
Yhanhk you!

Below are the screenshots of my configuration. Is there any setting that could optimize Waters MSe data so that DIA-NN can successfully generate the output files?


![Image](https://github.com/user-attachments/assets/64145516-e19c-438b-923c-3b3b48cf30e3)
![Image](https://github.com/user-attachments/assets/b5bc69c1-23a2-463a-b6ae-a18f1747c3cd)
![Image](https://github.com/user-attachments/assets/9123ee69-59af-425e-a47d-deae117c52f5)

**Comment by vdemichev** [comment]
I cannot help here unfortunately. Can only suggest trying to analyse with a small public lib - just to see if it works.

**Comment by mrosario01** [comment]
Thank you for all the help. I would like that, yes. If you could explain
the best workflow, I would be very grateful to you.

Tks again!😀

Em ter., 10 de mar. de 2026, 16:21, Vadim Demichev ***@***.***>
escreveu:

**Comment by mrosario01** [comment]
I will try  in other  PC and  with  public  library.

Thank you!

Em ter., 10 de mar. de 2026, 18:39, Michelle Barbosa Rosário <
***@***.***> escreveu:

**Comment by vdemichev** [comment]
Just any small lib matching the sample you can find.

**Comment by mrosario01** [comment]
This spectral library was generated from a FASTA file. Do you recommend generating it from a reduced FASTA, or from my own samples (in this case, one liver sample and one muscle sample, which are the two tissues I use)?

**Comment by vdemichev** [comment]
Any empirical library from any K-GG experiment on PRIDE :)

**Comment by mrosario01** [comment]
Thank you!

**Comment by matrvidal** [comment]
Hi @mrosario01 , did you manage to get it working on MSe data after the sugestions?
